src/module/item/affliction/document.ts

```diff
@@ -1,26 +1,32 @@
 import { ActorPF2e } from "@actor";
 import { AbstractEffectPF2e, EffectBadge } from "@item/abstract-effect/index.ts";
 import { UserPF2e } from "@module/user/index.ts";
+import { ItemSummaryData } from "@item/data/index.ts";
+import {
+    AfflictionOnset,
+    AfflictionDuration,
+    AfflictionFlags,
+    AfflictionSource,
+    AfflictionSystemData,
+} from "./data.ts";
+import { AfflictionDamageTemplate, DamagePF2e, DamageRollContext, BaseDamageData } from "@system/damage/index.ts";
 import { createDamageFormula, parseTermsFromSimpleFormula } from "@system/damage/formula.ts";
-import { AfflictionDamageTemplate, BaseDamageData, DamagePF2e, DamageRollContext } from "@system/damage/index.ts";
 import { DamageRoll } from "@system/damage/roll.ts";
-import { ErrorPF2e } from "@util";
-import { AfflictionFlags, AfflictionSource, AfflictionSystemData } from "./data.ts";
 
 class AfflictionPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends AbstractEffectPF2e<TParent> {
-    constructor(source: object, context?: DocumentConstructionContext<TParent>) {
-        super(source, context);
-        if (BUILD_MODE === "production") {
-            throw ErrorPF2e("Affliction items are not available in production builds");
-        }
-    }
+    static DURATION_UNITS: Readonly<Record<string, number>> = {
```

Comment:
Given this seems to be a copy of the effect one, I think we outta move this to the abstract effect superclass.